extension-style/src/main/java/com/mapbox/maps/extension/style/layers/Layer.kt

```diff
@@ -119,7 +119,7 @@ abstract class Layer : StyleContract.StyleLayerExtension {
 
   // Layer Properties
 
-  private fun getCachedLayerProperties(): Value {
+  internal fun getCachedLayerProperties(): Value {
```

Comment:
Is it needed for tests only? Could add `@Visibility` annotation then I guess

Reply:
no, it is used for the `Layer.bindPersistentlyTo` extension function